accessibility-checker-engine/src/v2/checker/accessibility/nls/index.ts

```diff
@@ -978,7 +978,7 @@ let a11yNls = {
         "Fail_combobox_popup_role_mismatch": "The value of the combobox 'aria-haspopup' attribute \"{0}\" does not match the 'role' value of the popup element \"{1}\""
     },    
     "combobox_focusable_elements": {
-        0: "Tabbable focus for the combobox must be allowed only on the text input, except when using a dialog popup",
+        0: "Tabable focus for the combobox must be allowed only on the text input, except when using a dialog popup",
```

Comment:
Tabbable was correct

Reply:
agree. This one shoud not have been changed. correcting back